lib/bemhtml/runtime/context.js

```diff
@@ -48,9 +46,7 @@ Context.prototype.isLast = function isLast() {
 };
 
 Context.prototype.generateId = function generateId() {
-  if (this._uniq === null)
-    this._uniq = utils.getUniq();
-  return this._uniq;
+  return utils.getUniq();
```

Comment:
To be compatible with old bem-xjst, it should actually generate id based on current context. Is it ok to generate random here, @veged ?